### DISCUSSION TITLE
Disassembler Memory viewer

### DISCUSSION THREAD

**Discussion by tiztrinity** [BartmanAbyss/vscode-amiga-debug#219] [General]
Hi
after dragging around the Disassembler Memory panel in VSCode (macos version), the content of the window disappeared; so I'm no more able to see the code disassembly.
Any idea on how to restore it back?
Thanks

**Comment by BartmanAbyss** [comment]
it worked for me in the command palette `ctrl+shift+p` > `run and debug: focus on disassembled memory viewer`

**Reply by tiztrinity** [reply]
Oh, yes. Worked!
Thanks!
